config/initializers/acts_as_taggable_on.rb

```diff
@@ -1,14 +1,6 @@
-# Rails.logger.debug "Running initializer in #{__FILE__}"
+# frozen_string_literal: true
 
-# Store the tags in the warehouse
-require Rails.root.join('config', 'initializers', 'db_warehouse')
-module ActsAsTaggableOn
-  class Tag < ::ActiveRecord::Base
-    establish_connection DB_WAREHOUSE
-  end
-  class Tagging < ::ActiveRecord::Base
-    establish_connection DB_WAREHOUSE
-  end
+ActsAsTaggableOn.setup do |config|
+  # Store the tags in the warehouse
+  config.base_class = 'GrdaWarehouseBase'
```

Comment:
lack of shared connection was leading to strange behavior in tests

Reply:
That'a a much cleaner fix. Thanks!